### ISSUE TITLE
auto config errors out after eslint-config-prettier@8.1

### ISSUE THREAD

**Issue by YoraiLevi** [davidjbradshaw/eslint-config-auto#18]
I am trying this auto config out, there seem to have been a recent change that might be causing this error (below)
https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

```
Error: Cannot read config file: D:\source\playZone\YoraiLevi.github.io\node_modules\eslint-config-prettier\@typescript-eslint.js     
Error: "prettier/@typescript-eslint" has been merged into "prettier" in eslint-config-prettier 8.0.0. See: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21
Referenced from: D:\source\playZone\YoraiLevi.github.io\node_modules\eslint-config-adjunct\rules\prettier.js
    at Object.<anonymous> (D:\source\playZone\YoraiLevi.github.io\node_modules\eslint-config-prettier\@typescript-eslint.js:1:69)    
    at Module._compile (C:\ProgramData\nvm\v14.15.1\node_modules\eslint\node_modules\v8-compile-cache\v8-compile-cache.js:192:30)    
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at module.exports (C:\ProgramData\nvm\v14.15.1\node_modules\eslint\node_modules\import-fresh\index.js:32:59)
    at loadJSConfigFile (C:\ProgramData\nvm\v14.15.1\node_modules\eslint\node_modules\@eslint\eslintrc\lib\config-array-factory.js:225:16)
    at loadConfigFile (C:\ProgramData\nvm\v14.15.1\node_modules\eslint\node_modules\@eslint\eslintrc\lib\config-array-factory.js:309:20)
    at ConfigArrayFactory._loadConfigData (C:\ProgramData\nvm\v14.15.1\node_modules\eslint\node_modules\@eslint\eslintrc\lib\config-array-factory.js:609:42)
```

`npm install eslint-config-prettier@7.2.0` resolves the issue

**Comment by davidjbradshaw** [comment]
I would just run with the last version for now.

i just broke my elbow falling off a bike, so might be a week or two before I can look at this

**Comment by davidjbradshaw** [comment]
This is now fixed and released in v0.7.0
